src/sentry/monitors/serializers.py

```diff
@@ -34,8 +86,23 @@ def get_attrs(
         ]
         environments = {env.id: env for env in Environment.objects.filter(id__in=env_ids)}
 
+        active_incidents = list(
+            MonitorIncident.objects.filter(
+                monitor_environment__in=item_list,
+                resolving_checkin=None,
+            )
+        )
+        serialized_incidents = {}
+        for active_incident, serialized_incident in zip(
+            active_incidents, serialize(active_incidents, user)
+        ):
+            serialized_incidents[active_incident.monitor_environment_id] = serialized_incident
```

Comment:
code style nit: could just be this yeah?

```suggestion
        serialized_incidents = {
            incident.monitor_environment_id: serialized_incident
            for incident, serialized_incident
            in zip(active_incidents, serialize(active_incidents, user))
        }
```

Reply:
ahh yes much cleaner